test/test.js

```diff
@@ -83,6 +85,24 @@ plantuml::test/fixtures/alice.puml[svg,role=sequence]
       const hash = rusha.createHash().update(`https://kroki.io/plantuml/svg/${encode(file)}`).digest('hex')
       expect(html).to.contain(`<img src=".asciidoctor/kroki/diag-${hash}.svg" alt="diagram">`)
     })
+    it('should create diagrams in imagesdir if kroki-fetch-diagram is set', function (done) {
+      const registry = asciidoctor.Extensions.create()
+      asciidoctorKroki.register(registry)
+      const file = `${__dirname}/fixtures/fetch/doc.adoc`
+      const doc = asciidoctor.convertFile(file, { extension_registry: registry, safe: 'unsafe' })
+      fs.unlinkSync(doc.getAttributes().outfile)
+      const imageLocation = path.join(doc.base_dir, doc.getAttributes().imagesdir)
+      fs.readdir(imageLocation, (err, files) => {
+        if (err) {
+          done(`Image location: ${imageLocation} does not exist`)
+        }
+        if (files.length === 1) {
```

Comment:
This should be an assertion:

```suggestion
        expect(files.length).to.equal(1)
```

Reply:
We can also use:

```suggestion
        expect(files).to.have.lengthOf(1)
```